src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+export const MovieCard: React.FC<Movie> = ({
```

Comment:
Create `Props` type or interface, you decide
https://mate-academy.github.io/fe-program/js/extra/react-typescript